app/rpc/rpc.go

```diff
@@ -53,7 +53,7 @@ func (m *Manager) routerInitializer(router *router.Router, netConnection *netada
 	for messageType := range handlers {
 		messageTypes = append(messageTypes, messageType)
 	}
-	incomingRoute, err := router.AddIncomingRoute(messageTypes)
+	incomingRoute, err := router.AddIncomingRoute("router", messageTypes)
```

Comment:
This should probably be "rpc - incoming" rather than "router"

Reply:
Changed to just "rpc router", the " - incoming" is added by AddIncomingRoute